src/flt-player.c

```diff
@@ -1178,7 +1178,7 @@ static void player_filter_load_patterns (void) {
 	scanner = g_scanner_new (&patterns_scanner_config);
 
 	for (i = TOKEN_STRING; i <= TOKEN_REGEXP; i++) {
-		g_scanner_add_symbol (scanner, mode_symbols[i - TOKEN_STRING],
+		g_scanner_scope_add_symbol (scanner, 0, mode_symbols[i - TOKEN_STRING],
```

Comment:
while you're at it, can you remove the line split (cleaning-up the style at the same time)?

Reply:
At least this file has only one change so that can be done there. otherwise don't worry.

Reply:
updated